### ISSUE TITLE
Add NEBULA to RSD

### ISSUE THREAD

**Issue by svenvanderburg** [esciencecenter-digital-skills/NEBULA#75]
It would be great if we can add NEBULA to the research software directory.

It helps us show the impact we have with open-source lesson material development.

**Comment by Peter9192** [comment]
I completely agree. I could link it under "impact" of the [EUCP project](https://research-software-directory.org/projects/european-climate-prediction-system) as this is originally derived from the storyboards we developed there, right?

**Comment by svenvanderburg** [comment]
Sounds good to me @Peter9192 !

**Comment by Peter9192** [comment]
I created an RSD page:
https://research-software-directory.org/software/nebula

I sent a maintainer link to @JaroCamphuijsen and @raar1
